.github/ISSUE_TEMPLATE/bug_report_v2.yaml

```diff
@@ -15,13 +15,13 @@
 name: 🐞 Bug Report
 description: File a bug report
 title: 'Title for the bug'
-labels: 'question, new'
+labels: 'new'
```

Comment:
Please keep the `question` label. Otherwise all new issues will be mirrored as `Bugs` instead of `Customer Issues` in b/.

Reply:
oh that's not good 😅 will revert